Formula/o/opencsg.rb

```diff
@@ -18,17 +18,19 @@ class Opencsg < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "b37ea3b9c6749f57cf8b91102f334a161b707a27e9c5d4f939ea946b445207e0"
   end
 
-  depends_on "qt" => :build
+  depends_on "cmake" => :build
   depends_on "glew"
 
-  def install
-    # Disable building examples
-    inreplace "opencsg.pro", "src example", "src"
+  # Upstream pr ref: https://github.com/floriankirsch/OpenCSG/pull/15
+  patch do
+    url "https://github.com/floriankirsch/OpenCSG/commit/2d618a8145388a4fe8da2475725ad9e84a683076.patch?full_index=1"
+    sha256 "1728803d091e44967b6bb788bdf1270d974d08da7e9417b200c40cc44b7b1900"
+  end
```

Comment:
It may be that this patch works only when building inside Homebrew because of our compiler shims.

Reply:
So changing back to `inreplace` might make more sense?